recipes/mfast/all/conandata.yml

```diff
@@ -1,24 +1,36 @@
 sources:
-  "1.2.1":
-    url: "https://github.com/objectcomputing/mFAST/archive/v1.2.1.tar.gz"
-    sha256: "e6ed19e629a33068b7ab5ebdbd860831f2f2b44b3b361068fbbbe975202a4851"
+  "1.2.2":
+    url: "https://github.com/objectcomputing/mFAST/archive/v1.2.2.tar.gz"
+    sha256: "a7686c4d2e3121d7c8dd7f43b70424b55d4a1f6843179a6b0709658be27f39cb"
   "cci.20200824":
     url: "https://github.com/objectcomputing/mFAST/archive/4546f9b2f26cc7801fa63c05a7354ecbcb04410a.zip"
     sha256: "ec2e34b0943066313db0a9cf2035ab8d5b28f9d46f0adfbb3ef9aa2a25280a4e"
-patches:
   "1.2.1":
-    - patch_file: "patches/00001-adapt-cmakelists.patch"
+    url: "https://github.com/objectcomputing/mFAST/archive/v1.2.1.tar.gz"
+    sha256: "e6ed19e629a33068b7ab5ebdbd860831f2f2b44b3b361068fbbbe975202a4851"
+patches:
+  "1.2.2":
+    - patch_file: "patches/cci.20200824-0001-adapt-cmakelists.patch"
+      base_path: "source_subfolder"
+    - patch_file: "patches/cci.20200824-0002-add-conan-libs-to-targets.patch"
       base_path: "source_subfolder"
-    - patch_file: "patches/00002-add-conan-libs-to-targets.patch"
+    - patch_file: "patches/cci.20200824-0003-remove-deprecated-boost-detail-endian.patch"
```

Comment:
Using the version makes them more difficult to resuse.... It looks odd seeing a "wrong" version here